packages/client/src/contracts/StorageNodeRegistry.ts

```diff
@@ -22,7 +21,7 @@ export interface StorageNodeMetadata {
 export class StorageNodeRegistry {
 
     private nodeRegistryContract?: NodeRegistryContract
-    private readonly nodeRegistryContractsReadonly: NodeRegistryContract[]
+    private readonly nodeRegistryContractsReadonly: NodeRegistryContract
```

Comment:
Rename (to singular)
```suggestion
    private readonly nodeRegistryContractReadonly: NodeRegistryContract
```